aws-s3-bucket.yml

```diff
@@ -159,6 +159,17 @@ provision:
     details: |
       Whether this bucket explicitly denies access to HTTP requests, in other words, the bucket only accepts
       requests sent through HTTPS if enabled.
+  - field_name: allowed_aws_vpc_id
+    type: string
+    details: |
```

Comment:
I think this makes in ambiguous as to whether the default configuration is secure. I think we should document what is does, and let users decide what level of security meets their requirements. How about something like:
```
The ID of a pre-created VPC Endpoint. When specified, the S3 bucket policy will only allow access from the specified VPC endpoint.
E.g: `vpc-01362976bd10dc099`.
To read about VPC endpoints see https://docs.aws.amazon.com/vpc/latest/privatelink/vpc-endpoints-s3.html
```